dspace/config/submission-forms.xml

```diff
@@ -585,191 +584,1193 @@
+                    <hint>Enter the URL of the project webpage</hint>
+                </field>
+            </row>
+            <row>
+                <field>
+                    <dc-schema>dc</dc-schema>
+                    <dc-element>identifier</dc-element>
+                    <label>Award number</label>
+                    <input-type>onebox</input-type>
+                    <hint>Enter the identifier of the project</hint>
+                </field>            
+                <field>
+                    <dc-schema>oaire</dc-schema>
+                    <dc-element>fundingStream</dc-element>
+                    <label>Name of the funding stream</label>
+                    <input-type>textarea</input-type>
+                    <hint>Enter the name of the funding stream of the project</hint>
+                </field>
+            </row>
+            <row>
+                <relation-field>
+                    <relationship-type>isFundingAgencyOfProject</relationship-type>
+                    <search-configuration>fundingAgencyConfiguration</search-configuration>
+                    <repeatable>false</repeatable>
+                    <label>Funding Agency</label>
+                    <hint>Add a Funding Agency</hint>
+                    <linked-metadata-field>
+                        <dc-schema>oaire</dc-schema>
+                        <dc-element>funder</dc-element>
+                        <dc-qualifier>name</dc-qualifier>
+                        <input-type>onebox</input-type>
+                    </linked-metadata-field>
+                    <required>One funding agency is required</required>
+                </relation-field>
+            </row>
+        </form>
+
+        <form name="openAIREOrganizationForm">
+            <row>
+                <field>
+                    <dc-schema>organization</dc-schema>
+                    <dc-element>legalName</dc-element>
+                    <label>Name</label>
+                    <input-type>onebox</input-type>
+                    <hint>Enter the name of the orgunit or organization</hint>
+                </field>
+            </row>
+            <row>
+                <field>
+                    <dc-schema>organization</dc-schema>
+                    <dc-element>identifier</dc-element>
+                    <dc-qualifier></dc-qualifier>
+                    <!-- An input-type of qualdrop_value MUST be marked as repeatable -->
+                    <repeatable>true</repeatable>
+                    <label>Organization identifier</label>
+                    <input-type value-pairs-name="openaire_organization_identifier_types">qualdrop_value</input-type>
+                    <hint>If the organization has any identifiers or codes associated with
+                        it, please enter the types and the actual numbers or codes.
+                    </hint>
+                    <required></required>
+                </field>
+            </row>
```

Comment:
Add new field to distinguish funding or not funding organizations:

            <row>
                <field>
                    <dc-schema>dc</dc-schema>
                    <dc-element>type</dc-element>
                    <label>Organization Type</label>
                   <input-type value-pairs-name="organization_types">dropdown</input-type>
                    <hint>Choose the attributes of the Organization</hint>
                    <required></required>
                </field>
            </row>




             <value-pairs value-pairs-name="organization_types" dc-term="type">
                <pair>	            
                    <displayed-value>N/A</displayed-value>
                    <stored-value></stored-value>	                
                </pair>

                <pair>                       
                    <displayed-value>Is a Funding Organization</displayed-value>
                    <stored-value>FundingOrganization</stored-value>
                </pair>          
           </value-pairs>